lib/rules/quotes.js

```diff
@@ -72,7 +85,16 @@ module.exports = function(context) {
                     context.report(node, "Strings must use " + settings.description + ".");
                 }
             }
+
+        } else if (node.type === "TemplateLiteral" && quoteOption !== "back") {
+            context.report(node, "Strings must use " + settings.description + ".");
         }
```

Comment:
We may need to think this through a bit. Does specifying "single" or "double" really mean you can't use template strings? My gut feeling is that this would be an unexpected restriction.